src/utils/openmc/tallies.py

```diff
@@ -581,6 +581,116 @@ def get_mesh_xml(self):
         return element
 
 
+class Trigger(object):
+
+    def __init__(self, trigger_type='rel_err', threshold=1E-2):
```

Comment:
The documentation says there is no default for the threshold, but here you assign a default of 1%. Does a default need to be assigned?